packages/react-native-sdk/jest-setup.ts

```diff
@@ -23,6 +23,11 @@ const mockedDevices = [
   },
 ];
 
+// Mock the notifee module using the mock provided by @notifee/react-native itself
+jest.mock('@notifee/react-native', () =>
+  require('@notifee/react-native/jest-mock'),
+);
```

Comment:
Excellent, can we try to mock this more elegantly via` __mocks__` folder in the root of the SDK?

I.e. create a file `__mocks__/@notifee/react-native.ts` with content:

```
import mock from '@notifee/react-native/jest-mock';

export default mock
```

Reply:
It's not about not following their docs. It's about having a more accessible and common way to do the same thing.

If it works I will raise a PR to their docs, because that's the way it should've been.

- the [__mocks__](https://jestjs.io/docs/manual-mocks#mocking-node-modules) folder is meant for auto mocking your modules.
- jest-setup.ts is meant for [global](https://jestjs.io/docs/configuration/#setupfiles-array) setup

So not elegance, yet correctness and using configuration as planned by Jest

Reply:
I tested it and It did not work..

I understand its a common way, but doesn't work notifee I guess when you import.. 

I suggest that you try it out if we really want to achieve this. This change did not work..

maybe there is something at play with the change in jest.config.ts

you can see https://github.com/invertase/notifee/tree/main/packages/react-native for details

Reply:
Fair enough, tnx for trying ;)